stack/keycloak/migration/03_add-roles.yml

```
@@ -0,0 +1,17 @@
+      name: reader
+      description: Example role with read access to application data
+      clientRole: true
+      clientId: local
+      composite: false
+
+  - addRole:
+      name: writer
+      description: Example role with write access to application data
+      clientRole: true
+      clientId: local
+      composite: false
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)